sdwmbar.c

```diff
@@ -89,6 +90,21 @@ getbatt(int *buff)
 	}
 }
 
+void
+getac(char *ac)
+{
+	int buff;
```

Comment:
Follow the code convention. Argument should be `char *buff`, swap the names around.